test/idrs.jl

```diff
@@ -57,4 +58,30 @@ end
     @test x_new ≈ x
 end
 
+@testset "Termination criterion" begin
+    for T in (Float32, Float64, ComplexF32, ComplexF64)
+        A = T[ 2 -1  0
+              -1  2 -1
+               0 -1  2]
+        n = size(A, 2)
+        b = ones(T, n)
+        x0 = A \ b
+        perturbation = T[(-1)^i for i in 1:n]
+
+        # If the initial residual is small and a small relative tolerance is used,
+        # many iterations are necessary
+        x = x0 + sqrt(eps(real(T))) * perturbation
```

Comment:
here you have a different initial residual size and you write “small” in both cases. Just something I stumbled upon

Reply:
Ah, I see. Thanks for noticing and explaining this. I modified these tests such that the same initial residual is used for the tests of the absolute and relative tolerances (0869558).